/**:
    ros__parameters:
        family: 36h11           # tag family name
        size: 0.024             # tag edge size in meter
        camera_device: "/dev/video4"  # camera device path or index (e.g., "/dev/video0" or "0")
        max_hamming: 0          # maximum allowed hamming distance (corrected bits)

        # see "apriltag.h" 'struct apriltag_detector' for more documentation on these optional parameters
        detector:
            threads: 1          # number of threads
            decimate: 2.0       # decimate resolution for quad detection
            blur: 0.0           # sigma of Gaussian blur for quad detection
            refine: True        # snap to strong gradients
            sharpening: 0.25    # sharpening of decoded images
            debug: False        # write additional debugging images to current working directory


        pose_estimation_method: "pnp"   # method for estimating the tag pose
